OpenGpxTracker-Watch/uk.lproj/Interface.strings

```diff
@@ -0,0 +1,60 @@
+/*
+ Interface.strings
+ OpenGpxTracker
+
+ Originally Translated to Ukrainian by Vadim Nekhai <vadimnekhai@gmail.com> on 13 January 2020
+ */
+
+/* Class = "WKInterfaceLabel"; text = "signal"; ObjectID = "036-aK-7ie"; */
+"036-aK-7ie.text" = "Сигнал";
+
+/* Class = "WKInterfaceLabel"; text = "File Name"; ObjectID = "19n-8m-sC5"; */
+"19n-8m-sC5.text" = "Ім’я файлу";
+
+/* Class = "WKInterfaceButton"; title = "View files"; ObjectID = "3QZ-4r-aMl"; */
+"3QZ-4r-aMl.title" = "Перегляд файлів";
+
+/* Class = "WKInterfaceLabel"; text = "speedValue"; ObjectID = "9yb-Lf-Jav"; */
+"9yb-Lf-Jav.text" = "Значення швидкості";
+
+/* Class = "WKInterfaceLabel"; text = "Sending"; ObjectID = "AyV-DL-c4U"; */
+"AyV-DL-c4U.text" = "Надсилання";
+
+/* Class = "WKInterfaceLabel"; text = "SPEED"; ObjectID = "HGe-sg-FuE"; */
+"HGe-sg-FuE.text" = "ШВИДКІСТЬ";
+
+/* Class = "WKInterfaceLabel"; text = "Alert Label"; ObjectID = "IdU-wH-bcW"; */
+"IdU-wH-bcW.text" = "Значення сповіщення";
+
+/* Class = "WKInterfaceLabel"; text = "File Name"; ObjectID = "Nkq-Do-muS"; */
+"Nkq-Do-muS.text" = "Ім’я файлу";
+
+/* Class = "WKInterfaceButton"; title = "Save"; ObjectID = "Yfa-t6-5Sk"; */
+"Yfa-t6-5Sk.title" = "Зберегти";
+
+/* Class = "WKInterfaceButton"; title = "Start Tracking"; ObjectID = "exN-t4-It8"; */
+"exN-t4-It8.title" = "Розпочати запис треку";
+
+/* Class = "WKInterfaceLabel"; text = "altValue"; ObjectID = "fLq-xh-Kj9"; */
+"fLq-xh-Kj9.text" = "Значення висоти";
+
+/* Class = "WKInterfaceButton"; title = "Reset"; ObjectID = "g9l-MU-PFV"; */
+"g9l-MU-PFV.title" = "Скинути";
+
+/* Class = "WKInterfaceLabel"; text = "0m"; ObjectID = "jLD-vq-fZz"; */
+"jLD-vq-fZz.text" = "0хв";
```

Comment:
Just realized that this could be about distance. Than translation should be `"0м"`. Let me re-check that and fix, a little bit later.

Reply:
This is the default label for the watch app's distance, so its basically 0 meters, abbr. as "0m"

Reply:
Haven't noticed this because looks like Imperial units were chosen earlier and I didn't dig into project structure at that moment.
<img width="378" alt="Screen Shot 2020-01-13 at 23 05 00" src="https://user-images.githubusercontent.com/645583/72295351-0f51c980-3660-11ea-9a40-947302a4669d.png">